frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -336,11 +336,11 @@ class Chart extends Component<Props, State> {
             toolTipValue: price,
             toolTipTop,
             toolTipLeft,
-            toolTipTime: time,
+            toolTipTime: time as number,
         });
     }
 
-    private renderDate = (date) => {
+    private renderDate = (date: string | number) => {
```

Comment:
seems like `number` alone should be enough? all invocations seem to be `number`

Reply:
fixed